src/common/data_compression.cpp

```diff
@@ -0,0 +1,56 @@
+// Copyright 2019 yuzu Emulator Project
+// Licensed under GPLv2 or any later version
+// Refer to the license.txt file included.
+
+#pragma once
+
+#include <lz4hc.h>
+
+#include "data_compression.h"
```

Comment:
```suggestion
#include "common/data_compression.h"
```

A better naming scheme here would also be `common/lz4_compression.h` (along with changing the cpp file). Making a singular cpp/h file for compression algorithms can get messy over time.